MediaBrowser.Api/UserService.cs

```diff
@@ -266,22 +266,36 @@ public class UserService : BaseApiService
             _authContext = authContext;
         }
 
+        /// <summary>
+        /// Gets the public available Users information
+        /// </summary>
+        /// <param name="request">The request.</param>
+        /// <returns>System.Object.</returns>
         public object Get(GetPublicUsers request)
         {
-            // If the startup wizard hasn't been completed then just return all users
-            if (!ServerConfigurationManager.Configuration.IsStartupWizardCompleted)
```

Comment:
why is this dropped?

Reply:
You're right, that's a good catch! I started implementing the method saying to myself "I have to add this checl before committing" and then I forgot...!

Reply:
I've introduced the check again! Thanks for finding this out :smile: